kits/bugpilot-ai/prompts/bugpilot-debugger_llmnode_user.md

```
@@ -0,0 +1,5 @@
+Programming Language:{{triggerNode_1.output.language}}
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)